pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -337,48 +337,69 @@ struct HdRprApiEnvironmentLight {
+        float lensShiftX = (m_cameraLens[0] + tile[0] + tileSizeX * 0.5 - 0.5) / tileSizeX;
+        float lensShiftY = (m_cameraLens[1] + tile[1] + tileSizeY * 0.5 - 0.5) / tileSizeY;
         RPR_ERROR_CHECK(camera->SetLensShift(lensShiftX, lensShiftY), "Failed to set lens shift");
-        if (m_Mode == RPR_CAMERA_MODE_PERSPECTIVE) {
+        if (m_mode == RPR_CAMERA_MODE_PERSPECTIVE) {
             RPR_ERROR_CHECK(camera->SetSensorSize(
-                m_SensorSize[0] * tileSizeX * (tileSizeY < tileSizeX ? tileSizeY / tileSizeX : 1),
-                m_SensorSize[1] * tileSizeY * (tileSizeX < tileSizeY ? tileSizeX / tileSizeY : 1)),
+                m_sensorSize[0] * tileSizeX * (tileSizeY < tileSizeX ? tileSizeY / tileSizeX : 1),
+                m_sensorSize[1] * tileSizeY * (tileSizeX < tileSizeY ? tileSizeX / tileSizeY : 1)),
                 "Failed to set sensor size");
         } 
-        else if (m_Mode == RPR_CAMERA_MODE_ORTHOGRAPHIC) {
-            RPR_ERROR_CHECK(camera->SetOrthoWidth(m_OrthoWidth * tileSizeX * (tileSizeY < tileSizeX ? tileSizeY / tileSizeX : 1)), "Failed to set ortho width");
-            RPR_ERROR_CHECK(camera->SetOrthoHeight(m_OrthoHeight * tileSizeY * (tileSizeX < tileSizeY ? tileSizeX / tileSizeY : 1)), "Failed to set ortho height");
-        }
-        else if (m_Mode == RPR_CAMERA_MODE_LATITUDE_LONGITUDE_360) {
+        else if (m_mode == RPR_CAMERA_MODE_ORTHOGRAPHIC) {
+            double shiftX = tile[0] + tileSizeX * 0.5 - 0.5;
+            double shiftY = tile[1] + tileSizeY * 0.5 - 0.5;
+            GfMatrix4f transform;
+            transform.SetIdentity().SetTranslate(GfVec3f(hdCamera->GetHorizontalAperture() * shiftX, hdCamera->GetVerticalAperture() * shiftY, 0));
+            transform *= m_Transform;
+            transform.Orthonormalize();
+            RPR_ERROR_CHECK(camera->SetTransform(transform.GetArray(), false), "Failed to set camera transform");
+            RPR_ERROR_CHECK(camera->SetOrthoWidth(m_orthoWidth * tileSizeX * (tileSizeY < tileSizeX ? tileSizeY / tileSizeX : 1)), "Failed to set ortho width");
+            RPR_ERROR_CHECK(camera->SetOrthoHeight(m_orthoHeight * tileSizeY * (tileSizeX < tileSizeY ? tileSizeX / tileSizeY : 1)), "Failed to set ortho height");
+            RPR_ERROR_CHECK(camera->SetLinearMotion(0.0f, 0.0f, 0.0f), "Failed to set camera linear motion");
+            RPR_ERROR_CHECK(camera->SetAngularMotion(1.0f, 0.0f, 0.0f, 0.0f), "Failed to set camera angular motion");
+        }
+        else if (m_mode == RPR_CAMERA_MODE_LATITUDE_LONGITUDE_360) {
             // do nothing
         }
 
     }
 private:
-    float m_CameraLens[2];
-    float m_SensorSize[2];
-    rpr_camera_mode m_Mode;
-    float m_OrthoWidth;
-    float m_OrthoHeight;
+    float m_cameraLens[2];
+    float m_sensorSize[2];
+    rpr_camera_mode m_mode;
+    float m_orthoWidth;
+    float m_orthoHeight;
+    GfMatrix4f m_Transform;
```

Comment:
Should m_Transform be renamed to m_transform  like it was done for other variables ??